build-system/tasks/visual-diff/index.js

```diff
@@ -813,9 +810,30 @@ async function installDependencies_() {
   percySnapshot = require('@percy/puppeteer');
   Percy = require('@percy/core');
 
-  const browserFetcher = puppeteer.createBrowserFetcher();
-  if (argv.noinstall) {
-    return browserFetcher;
+/**
+ */
+function setupCleanup_() {
+  process.on('exit', cleanup_);
+  process.on('SIGINT', cleanup_);
+  process.on('uncaughtException', cleanup_);
+  process.on('unhandledRejection', cleanup_);
+}
+
+/**
+ * @return {Promise<void>}
+ */
+async function exitPercyAgent_() {
+  if (percyAgentProcess_ && !percyAgentProcess_.killed) {
+    let resolver;
+    const percyAgentExited_ = new Promise((resolverIn) => {
+      resolver = resolverIn;
+    });
+    percyAgentProcess_.on('exit', () => {
+      resolver();
+    });
+    // Explicitly exit the process by "Ctrl+C"-ing it.
+    await percyAgentProcess_.kill('SIGINT');
+    await percyAgentExited_;
```

Comment:
These functions were deleted recently, so this might be the result of a bad merge. I'd ask @danielrozenberg for a careful review of this file.

Reply:
Great catch!

Reply:
Is this file no longer part of this PR?

Reply:
Yeah, I reverted the whole file. I intend to fix types in the tasks/visual-diff directory all in one separate PR